mm2src/coins/lp_coins.rs

```diff
@@ -668,6 +673,22 @@ impl From<keys::Error> for TransactionErr {
     fn from(e: keys::Error) -> Self { TransactionErr::Plain(e.to_string()) }
 }
 
+impl From<NftAssocTypesError> for TransactionErr {
+    fn from(e: NftAssocTypesError) -> Self { TransactionErr::NftAssocTypesError(e.to_string()) }
+}
+
+impl From<NumConversError> for TransactionErr {
+    fn from(e: NumConversError) -> Self { TransactionErr::NumConversError(e) }
+}
```

Comment:
same here and other places too

Reply:
started to wrapping it into TransactionErr::Plain(ERRL!())  so we can have files path in error message, as I see we use Result instead of MmResult in some SwapOpsV2 methods.